### PR TITLE
ELEMENTS-1463: add support for webui preview

### PR DESCRIPTION
**Pull request by richardsd**
This PR depends on:
- https://github.com/nuxeo/ui-team-gh-actions/pull/5 (updates the trigger-workflow action to include an output message)
- https://github.com/nuxeo/nuxeo-web-ui/pull/1436 to be merged first, to update the `nuxeo-web-ui` branch

### CODE REVIEW

**Review by semisse — DISMISSED** [review]

File: .github/workflows/cleanup.yaml [hunk 1]
```diff
@@ -34,3 +34,24 @@ jobs:
...
+    if: (github.event.action == 'unlabeled' && github.event.label.name == 'webui-preview') || contains(github.event.pull_request.labels.*.name, 'webui-preview')
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Prepare environment
+        run: |
+          echo "BRANCH_NAME=${GITHUB_HEAD_REF##*/}" >> $GITHUB_ENV
+
+      - name: Web UI preview
+        uses: nuxeo/ui-team-gh-actions/trigger-workflow@bcc698511e6d9ce90ea120a6ee2cdde82079cde4
```
Comment (richardsd) [inline .github/workflows/cleanup.yaml]: Once https://github.com/nuxeo/ui-team-gh-actions/pull/5 is merged, we need to update this reference to the `trigger-workflow`

File: .github/workflows/cleanup.yaml [hunk 2]
```diff
@@ -34,3 +34,24 @@ jobs:
...
+        with:
+          owner: nuxeo
+          repo: nuxeo-web-ui
+          workflow-id: remote-clean.yaml
```
Comment (richardsd) [inline .github/workflows/cleanup.yaml]: Once https://github.com/nuxeo/nuxeo-web-ui/pull/1436 is merged, we need to update any reference to the workflows `remote-{clean,preview}` to just `{clean,preview}`.

File: .github/workflows/preview.yaml [hunk 3]
```diff
@@ -112,5 +112,39 @@ jobs:
...
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Prepare environment
+        run: |
+          echo "BRANCH_NAME=${GITHUB_HEAD_REF##*/}" >> $GITHUB_ENV
+
+      - name: Web UI preview
+        id: main
+        uses: nuxeo/ui-team-gh-actions/trigger-workflow@bcc698511e6d9ce90ea120a6ee2cdde82079cde4
```
Comment (richardsd) [inline .github/workflows/preview.yaml]: Same as above.

File: .github/workflows/preview.yaml [hunk 4]
```diff
@@ -112,5 +112,39 @@ jobs:
...
+        with:
+          owner: nuxeo
+          repo: nuxeo-web-ui
+          workflow-id: remote-preview.yaml
```
Comment (richardsd) [inline .github/workflows/preview.yaml]: Same as above.

File: .github/workflows/preview.yaml [hunk 5]
```diff
@@ -112,5 +112,39 @@ jobs:
...
+          access-token: ${{ secrets.WEBUI_ACCESS_TOKEN_PERSONAL }}
+          branch-name: WEBUI-784-webui-elements-preview
```
Comment (richardsd) [inline .github/workflows/preview.yaml]: Branch name should also be updated to `maintenance-3.0.x`.

**Review by Gabez0r — COMMENTED** [review]
LGTM

**Comment by richardsd** [comment]
PR updated with correct references for Web UI `maintenance-3.0.x` and for  UI actions (trigger-workflow with support for workflow output).

**Review by Gabez0r — APPROVED** [review]

**Comment by richardsd** [comment]
Cross-repo check is failing due to a11y checks not properly running in node v10 (see https://github.com/nuxeo/nuxeo-web-ui/pull/1456).
(not a problem for the actual PR content, is it address in other PR)

**Review by semisse — APPROVED** [review]

**Pull request merged**